resolver/GlobalPass.cc

```diff
@@ -82,7 +82,12 @@ bool resolveTypeMember(core::GlobalState &gs, core::ClassOrModuleRef parent, cor
     if (!sym.data(gs)->derivesFrom(gs, core::Symbols::Class()) && myVariance != parentVariance &&
         myVariance != core::Variance::Invariant) {
         if (auto e = gs.beginError(myTypeMember.data(gs)->loc(), core::errors::Resolver::ParentVarianceMismatch)) {
-            e.setHeader("Type variance mismatch with parent `{}`", parent.show(gs));
+            e.setHeader(
+                "Type variance mismatch with parent `{}`; child `{}` should be `{}` or invariant, but it is `{}`",
+                parent.show(gs), sym.show(gs), core::Polarities::showVariance(parentVariance),
+                core::Polarities::showVariance(myVariance));
+            e.addErrorLine(parentTypeMember.data(gs)->loc(), "Variance for parent `{}` declared here", parent.show(gs));
```

Comment:
This is a bit of a nit, but technically this error line points not to the variance, but to the type member itself. Maybe this could say simply, ``"Parent `{}` declared here"``.